### DISCUSSION TITLE
Miniplayer or audio autostart when I quit

### DISCUSSION THREAD

**Discussion by dotvav** [ReVanced/revanced-discussions#1070] [Question]
Hey. After I play a video and it stops (it reaches its end, I pause or I stop it), if I go back to the Android home screen, or if I turn my phone's screen off, then Youtube will restart playing it. Sometimes the audio only, sometimes with the video in the miniplayer. It seems like it's happening only when I don't have a bluetooth audio device connected.

I am using revanced 1.21.2 on YT 19.16.39.
[selected_patches_2024-09-07_20_14_09.json](https://github.com/user-attachments/files/16919415/selected_patches_2024-09-07_20_14_09.json)

I cannot find a configuration option that seems related. Is this a known issue, is there a fix? I cannot find another discussion related to this, but maybe I am searching the wrong words...

**Comment by oSumAtrIX** [comment]
You touched the spoof client settings which came with a warning. Reset to default.
